src/styles/index.scss

```diff
@@ -1,3 +1,37 @@
-body {
+@import './header';
+@import './card';
+@import './colors';
+@import './stars';
+@import './page-card';
+
+* {
   margin: 0;
+  box-sizing: border-box;
+
+  transition: all 300ms;
+}
+
+.main {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+
+  grid-gap: 48px 46px;
+
+  padding: 50px 40;
```

Comment:
The padding shorthand property seems to be missing a unit for the second value. It should be specified with a unit like 'px', '%', etc.